aws/templates/id/id_contenthub.ftl

```diff
@@ -47,6 +50,9 @@
     ]
 [/#function]
 
+
+[#assign CONTENTHUB_NODE_COMPONENT_TYPE = "contentnode"]
+
 [#assign componentConfiguration +=
     {
         CONTENTHUB_NODE_COMPONENT_TYPE : [
```

Comment:
Index needs provider in value I think - its the specific component config IF you are using AWS.

Don;'t know if we would ever end up with the same component type in two different providers in a single solution, but for consistency anyting related to a specific provider should have AWS_ on the front I think?

Reply:
So the solution would be the point that the provider component is selected? And the component is the codeontap definition of a function for a provider?